eng/pipelines/vmr-validation.yml

```diff
@@ -0,0 +1,51 @@
+### This pipeline validates new changes of the VMR (https://github.com/dotnet/dotnet)
+### VMR is cloned and updates are applied locally
+
+parameters:
+- name: vmrBranch
+  displayName: dotnet/dotnet branch to use
+  type: string
+  default: $(System.PullRequest.TargetBranch)
+
+trigger: none
+
+pr:
+  branches:
+    include:
+    - main
+    - release/8*
```

Comment:
This should not be scoped to 8* it should just be *.  We shouldn't have to update this once we start work on 9, etc.

Reply:
Right, the 7 branches won't have the YAML so it won't trigger.